images/kibana/docker-compose.production.yml

```diff
@@ -0,0 +1,6 @@
+version: "2"
```

Comment:
these are unit tests for the images, which were used with automated builds on Docker Hub, and are now used by the images/build-images.sh script. During development phases testing the images only through the AMP core stack would be a lot of pain.